src/tm2/tm-device.cpp

```diff
@@ -563,6 +589,7 @@ namespace librealsense
             case RS2_STREAM_POSE:
             {
                 auto tm_profile = _tm_supported_profiles.sixDof[stream_index];
+                tm_profile.mode = _tm_mode;
```

Comment:
These options are for Pose stream only. Is it possible to configure those as N/A if the user request raw IMU and/or FE streams without Pose ?

Reply:
The `mode` is already set to reasonable defaults, as part of the profiles, for all stream types (including pose).  For pose I override it rather than create a permutation of all possible `mode`s in the profiles and then have to select the profile that is asked for.  `mode` doesn't quite fit the profile concept, but it's the only way we have to pass these parameters for now.